### DISCUSSION TITLE
Plans for Astro 5?

### DISCUSSION THREAD

**Discussion by Mood93** [incluud/accessible-astro-starter#108] [General]
Love this project! Any plans or timeline for upgrading to Astro 5?

**Comment by markteekman** [comment]
@Mood93 most certainly! I will be working on that soon, looking into the upgrade guide as we speak 🙂

**Comment by markteekman** [comment]
@Mood93 since Astro is awesome as always, it was a quick upgrade: https://github.com/markteekman/accessible-astro-starter/releases/tag/v3.2.3 :D
